src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim());
+
+  declarations.forEach((declaration) => {
+    const [key, value] = declaration.split(':').map((part) => part.trim());
+
+    styleObject[key] = value;
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Here, you can use `reduce` to transform the array of declarations directly into an object, which is more declarative and avoids the manual object construction inside the `forEach`.